frontend/src/MakerApp.tsx

```diff
@@ -187,18 +185,6 @@ export default function App() {
     );
 }
 
-function priceToDisplay(hasEnteredPrice: boolean, orderPrice: string, priceInfo: PriceInfo | null) {
-    if (!priceInfo) {
-        return format("0");
-    }
-
-    if (!hasEnteredPrice) {
-        return format((priceInfo.ask * SPREAD).toString());
-    }
-
-    return format(orderPrice);
```

Comment:
We are no longer calling `format` on the price now, is that intentional?

Reply:
fixed (as per the fixup commit) - added format when the button refreshes.